chain-vote/src/lib.rs

```diff
@@ -250,7 +289,10 @@ pub fn result(
             }
         }
     }
-    TallyResult { votes }
+    TallyResult {
+        votes,
+        options: (0..options as u8),
```

Comment:
@mzabaluev I'm not really happy about this conversion (it comes as an `usize`). Any suggestion?

Reply:
`TallyState` tabulates per proposal, so it's appropriate. May want to add an assertion, though.

Reply:
I have changed the options range type to `usize` for consistency throughout chain-vote.
The limits on the range of choices should be applied in mockchain.